### ISSUE TITLE
Unhalndeld exception, screenshot attached.

### ISSUE THREAD

**Issue by cutmeat** [tzachshabtay/MonoAGS#318]
![grafik](https://user-images.githubusercontent.com/31467653/60755726-197d7600-9ff4-11e9-9139-adc30bb8b891.png)

Heyhey, after successfully compiling for Desktop on Windows I tried to start the Editor.exe. I got the attached exception.
Best regards

**Comment by tzachshabtay** [comment]
Thanks for the report! Have you tried running DemoQuest.Desktop directly (without the editor)?

The editor is in very early stages right now, currently in order to load the game you have to first compile it (in debug mode), if you haven't I think you'll get this exception.

**Comment by cutmeat** [comment]
Hey Tzach,
yes, I have tried the game, which worked. When you say debug mode, you mean choose the Debug solution for compilation? I can try that when I get home. I think I used the "Desktop" option.

**Comment by tzachshabtay** [comment]
Yes, that's what I meant. The "DesktopRelease" option is meant to be optimized for actual deployment, but "Debug" should be used for actual development, and the editor is currently wired to look for a debug build.
Thanks for trying it out.

**Comment by cutmeat** [comment]
Hey Tzach,
you were right and I got the Demo game to open after building the Debug solution. Thanks!
